src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7621,6 +7621,16 @@ public sealed partial class StringBuilder : System.Runtime.Serialization.ISerial
         void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public override string ToString() { throw null; }
         public string ToString(int startIndex, int length) { throw null; }
+
+        // foreach over ReadOnlyMemory<char> support (IEnumerable-like interface)
```

Comment:
This comment can/should be deleted from the ref, as we're trying to get to a state where these files are easily machine re-generatable.